tools/NodeConfigCompare/configuration/cloud_formation_template.yaml

```diff
@@ -281,6 +282,57 @@ Resources:
         ServerSideEncryptionConfiguration:
           - ServerSideEncryptionByDefault:
               SSEAlgorithm: AES256
+      LoggingConfiguration:
+        DestinationBucketName: !Ref LoggingBucket
+        LogFilePrefix: NodeConfigCompare
+  
+  S3BucketPolicyConfigTestingBucket:
```

Comment:
Rename this resource to `S3ConfigTestingBucketPolicy`

Reply:
👍 , will rename